easybuild/easyconfigs/a/Autotools/Autotools-20150119-GCC-4.9.2.eb

```diff
@@ -14,4 +14,4 @@ dependencies = [
     ('libtool', '2.4.5'), # 20150119
 ]
 
-moduleclass = 'devel'
+moduleclass = 'bundle'
```

Comment:
hm, is this really a bundle? just asking; because in that case also bodeps in possibly in the same group!